include/wx/msdos/apptrait.h

```diff
@@ -23,7 +23,9 @@ class wxGUIAppTraits : public wxGUIAppTraitsBase
 {
 public:
     virtual wxEventLoopBase *CreateEventLoop();
-    virtual wxPortId GetToolkitVersion(int *majVer = NULL, int *minVer = NULL) const;
+    wxPortId GetToolkitVersion(int *majVer = NULL,
```

Comment:
As far as I remember, the MS-DOS port is obsolete :wink: . 
I don't want to seem being stubborn, but what about removing it?